.circleci/config.yml

```diff
@@ -123,26 +123,26 @@ jobs:
           name: Chromatic
           command: yarn chromatic
 
-  # release:
-  #   docker:
-  #     - image: circleci/node:14.15.0-browsers
-  #   working_directory: ~/circleci-f36-build
-  #   steps:
-  #     - checkout
-  #     - attach_workspace:
-  #         at: ~/circleci-f36-build
-  #     - yarn_install
-  #     - add_ssh_keys:
-  #         fingerprints:
-  #           - '2b:9f:13:79:d8:7b:f5:38:43:8f:3d:a8:7d:53:22:b2'
-  #     - run: git config --global user.email ${GITHUB_EMAIL}
-  #     - run: git config --global user.name ${GITHUB_USER}
-  #     - run:
-  #         name: Build
-  #         command: yarn build
-  #     - run:
-  #         name: Deploy
-  #         command: yarn lerna publish --no-private --exact --canary --no-git-tag-version --no-push --preid beta --dist-tag beta --force-publish --no-changelog --yes
+  release:
+    docker:
+      - image: circleci/node:14.15.0-browsers
+    working_directory: ~/circleci-f36-build
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/circleci-f36-build
+      - yarn_install
+      - add_ssh_keys:
+          fingerprints:
+            - '2b:9f:13:79:d8:7b:f5:38:43:8f:3d:a8:7d:53:22:b2'
+      - run: git config --global user.email ${GITHUB_EMAIL}
+      - run: git config --global user.name ${GITHUB_USER}
+      - run:
+          name: Build
+          command: yarn build -- --no-cache
```

Comment:
do we need that extra `--`? or is it a leftover?

Reply:
I think it's a leftover, I'm going to remove it